events/blocks/adobe-connect/adobe-connect.js

```diff
@@ -0,0 +1,145 @@
+import { LIBS, getMetadata } from '../../scripts/utils.js';
+
+const { createTag } = await import(`${LIBS}/utils/utils.js`);
+
+function addParams(searchParams, params, key, value) {
+  if (params.has(key)) {
+    searchParams[key] = params.get(key);
+  } else {
+    searchParams[key] = value;
+  }
+}
+
+function addSearchParams(url, searchParams) {
+  const urlObj = new URL(url);
```

Comment:
Looks like the URL comes from user input, I think it would be good to have a try catch for this URL construction and then fail gracefully outside the block.